### PR TITLE
Task jd/j109 add markup for live page

### PR DESCRIPTION
**Pull request by JuliaDzyuba**

### CODE REVIEW

**Comment by JuliaDzyuba** [comment]
editing episode markup will be in a different branch (ticket 121)

File: frontend/src/components/podcast-live/podcast-live.tsx [hunk 1]
```diff
@@ -31,32 +48,54 @@ const PodcastLive: React.FC = () => {
     dispatch(recordActions.stopRecord());
   };
 
+  const handleCreateComment = (payload: CommentFormCreatePayload): void => {
+    dispatch(episodeActions.createComment(payload));
+    alert(payload.text);
```
Comment (Denidzio) [inline frontend/src/components/podcast-live/podcast-live.tsx]: Why did you use 'alert' here?

File: frontend/src/components/app/app.tsx [hunk 2]
```diff
@@ -79,6 +79,11 @@ const App: React.FC = () => {
           component={Episode}
           exact
         />
+        <AuthPublicRouter
```
Comment (ghost) [inline frontend/src/components/app/app.tsx:77]: remove an old route for podcast live

File: frontend/src/components/common/common.ts [hunk 3]
```diff
@@ -14,6 +14,9 @@ export { default as Loader } from './loader/loader';
 export { default as Select } from './select/select';
 export { default as Datepiker } from './datepicker/datepicker';
 export { default as Table } from './table/table';
+export { default as Comment } from './comment/comment';
```
Comment (ghost) [inline frontend/src/components/common/common.ts]:
```suggestion
export { default as CommentItem } from './comment-item/comment-item';
```

File: frontend/src/components/episode/episode.tsx [hunk 4]
```diff
@@ -60,6 +61,9 @@ const Episode: React.FC = () => {
               <p className={styles.description}>{episode.description}</p>
               <p className={styles.type}>Type: {episode.type}</p>
               <p className={styles.type}>Status: {episode.status}</p>
+              {
+                isStaging && isOwner && <Button label={'Start Live'} href={`${AppRoute.EPISODES}/${id}${AppRoute.LIVE}`}/>
```
Comment (ghost) [inline frontend/src/components/episode/episode.tsx]:
```suggestion
                isStaging && isOwner && <Button label="Start Live" href={`${AppRoute.EPISODES}/${id}${AppRoute.LIVE}`}/>
```

File: frontend/src/components/app/app.tsx [hunk 5]
```diff
@@ -79,6 +79,11 @@ const App: React.FC = () => {
...
+          path={AppRoute.EPISODES_$ID_LIVE}
+          component={PodcastLive}
```
Comment (ghost) [inline frontend/src/components/app/app.tsx]:
```suggestion
          component={EpisodeLive}
```

File: frontend/src/components/podcast-live/podcast-live.tsx [hunk 6]
```diff
@@ -1,15 +1,32 @@
...
 import Button from 'components/common/button/button';
 import styles from './styles.module.scss';
+import { CreateCommentForm, CommentsList } from 'components/common/common';
+import { CommentFormCreatePayload } from 'common/types/types';
+
+type PageParams = {
+  id: string;
+};
 
 const PodcastLive: React.FC = () => {
```
Comment (ghost) [inline frontend/src/components/podcast-live/podcast-live.tsx]:
```suggestion
const EpisodeLive: React.FC = () => {
```

File: frontend/src/components/podcast-live/podcast-live.tsx [hunk 7]
```diff
@@ -1,15 +1,32 @@
-import { useAppSelector, useDispatch, useEffect } from 'hooks/hooks';
```
Comment (ghost) [inline frontend/src/components/podcast-live/podcast-live.tsx:1]: rename file to `frontend/src/components/episode-live/episode-live.tsx`

**Review by ghost — CHANGES_REQUESTED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**